modules/claims_api/app/controllers/claims_api/v1/forms/power_of_attorney_controller.rb

```diff
@@ -223,6 +224,25 @@ def build_representative_info(poa_code)
             }
           end
         end
+
+        def check_file_number_exists!
+          ssn = target_veteran.ssn
+
+          begin
+            response = bgs_service.people.find_by_ssn(ssn) # rubocop:disable Rails/DynamicFindBy
+            unless response && response[:file_nbr].present?
+              error_message = 'Unable to locate Veteran FileNumber. '\
+                              'We apologize for the inconvenience. '\
+                              'Teams are actively working on getting this fixed. '\
```

Comment:
Do we know this is the case? I would be fine just shortening the error message to "Unable to locate Veteran file number. Please contact the National Help Desk at 1-800-983-0937 to have a ticket created."